### ISSUE TITLE
SVG-mode - "height: auto" caused IE resizing problem.

### ISSUE THREAD

**Issue by ABrumm** [wojtekmaj/react-pdf#251]
Hey Wojciech,

on a current project i discovered a problem with the auto-height in IE11 on SVG-Mode.
The same problem can be repeated on your test-page with the IE11: 

http://projekty.wojtekmaj.pl/react-pdf/test/

When i remove the auto-height in the webinspector, i got a fully sized SVG-PDF.

My Question is, for which reason do you set the auto-height in PageSVG.jsx?

```
renderSVG = () => {
    const { page } = this.context;

    this.renderer = page.getOperatorList();

    return this.renderer
      .then((operatorList) => {
        const svgGfx = new PDFJS.SVGGraphics(page.commonObjs, page.objs);
        this.renderer = svgGfx.getSVG(operatorList, this.viewport)
          .then((svg) => {
            svg.style.maxWidth = '100%';
            **svg.style.height = 'auto';**
            this.setState({ svg }, () => this.onRenderSuccess());
          })
          .catch(this.onRenderError);
      })
      .catch(this.onRenderError);
  }
```

In other browsers like Chrome, Safari or Firefox it could be removed without negative side effects on the PDF.

Hope to hear from you sone

**Comment by wojtekmaj** [comment]
This is because `max-width: 100%` above would cause the page to shrink in size if it does not fit in the container. Without this, `svg width` attribute would take over and make the page look squeezed. Without this code though, the page which serves as a white background would not always be covering the entire rendered page. Here's what it would look like without these two lines, in some cases:

![obraz](https://user-images.githubusercontent.com/5426427/43657228-74623cb6-9755-11e8-988b-ea115c4b04a1.png)

The more I think about it though, it is none of `react-pdf`'s business actually, so it would make sense to remove these two styles.

**Comment by ABrumm** [comment]
Did you try the svg Mode in IE11 Wojciech? The biggest Problem i get in my Case and on your test page is not the width, the Max-width 100% is totally ok. The Auto-height did not auto fit the page Container for the pdf. The only Way to Resize is to style the svg Element, which destroyed the renderAnnotation possibility.

**Comment by wojtekmaj** [comment]
Yes, I tried SVG in IE11. Indeed it looks bad. `max-width: 100%` without setting `height: auto` will cause SVG to have bad proportions in some cases, though.

I tried removing `height` attribute from the SVG so that I wouldn't need to set `height: auto` by CSS, but that makes the issue even worse as it still looks the same on IE, only now you don't have default height attribute easily available.

The only thing that was sensible to do was to remove the CSS altogether to make it work consistently with canvas. This will be fixed in 4.0.0.
